### ISSUE TITLE
Cannot install as another account

### ISSUE THREAD

**Issue by ba-johnson** [cjmurph/PmsService#40]
Hi there,

Since initially installing under my user context, I'm now unable to successfully get the installation to work under a specific service account context I've created for this application. The installation presents with the usual 'Unable to connect to service' when run under the context of the Plex service account I've created. I've tried adding that account as a local admin (it's currently a domain user) but I suspect that perhaps there are some settings which the MSI uninstaller isn't removing from when I initially installed it in my own context. Is there anything which you can think of which might be causing some stale settings on changing the context between different users? Ideally, I want to be able to run it in the context of a service account which will have its interactive logon rights disabled and as low privilege level as possible.

Thanks for your help!

**Comment by ba-johnson** [comment]
<bump>

**Comment by cjmurph** [comment]
Hi,

So you're saying the tray application can't connect to the service? You need to make sure the ports match on the service and the tray application.

**Comment by ba-johnson** [comment]
No that's not what I'm saying. I initially tried out the installation using my own domain account and since then, I have tried to uninstall the application and reinstall using another account's credentials. However, whilst the installation succeeds and the service starts in the Plex Service account's context, the service tray shows as unable to connect and the web interface for Plex also does not work. Do you store any settings in HKLM or somewhere which may be missed during an installation that I need to manually clean up in order to get it to work?

**Comment by cjmurph** [comment]
No I don't. Does Plex start ok if you log in as that user account and start it?

**Comment by ba-johnson** [comment]
I'm going to give that a shot now. However, in the meantime, the log file outputs the following when I do try to start the service from my session but in the service account's context:

3/1/2019 8:31:02 AM: Plex Service Stopped
3/1/2019 8:31:04 AM: Plex executable found at C:\Program Files (x86)\Plex\Plex Media Server\Plex Media Server.exe
3/1/2019 8:31:04 AM: Successfully set the 'FirstRun' registry key to 0
3/1/2019 8:31:04 AM: Attempting to start Plex
3/1/2019 8:31:04 AM: Plex Media Server version is 1.14.1.5488. Can use startup argument.
3/1/2019 8:31:04 AM: Plex Service Started
3/1/2019 8:31:05 AM: Plex Media Server Started.
3/1/2019 8:31:06 AM: Plex Media Server has stopped!
3/1/2019 8:31:06 AM: Killing supporting processes.
3/1/2019 8:31:06 AM: Looking for Plex DLNA Server
3/1/2019 8:31:07 AM: 0 instances of Plex DLNA Server found
3/1/2019 8:31:07 AM: Looking for PlexScriptHost
3/1/2019 8:31:07 AM: 0 instances of PlexScriptHost found
3/1/2019 8:31:07 AM: Looking for PlexTranscoder
3/1/2019 8:31:07 AM: 0 instances of PlexTranscoder found
3/1/2019 8:31:07 AM: Looking for PlexNewTranscoder
3/1/2019 8:31:07 AM: 0 instances of PlexNewTranscoder found
3/1/2019 8:31:07 AM: Looking for Plex Media Scanner
3/1/2019 8:31:07 AM: 0 instances of Plex Media Scanner found
3/1/2019 8:31:07 AM: Looking for PlexRelay
3/1/2019 8:31:07 AM: 0 instances of PlexRelay found
3/1/2019 8:31:07 AM: Looking for EasyAudioEncoder
3/1/2019 8:31:07 AM: 0 instances of EasyAudioEncoder found
3/1/2019 8:31:07 AM: Looking for Plex Tuner Service
3/1/2019 8:31:07 AM: 0 instances of Plex Tuner Service found
3/1/2019 8:31:07 AM: Waiting 300 seconds before re-starting the Plex process.

**Comment by ba-johnson** [comment]
OK... I've resolved it using ProcMon to check what files are being accessed during the service start. Basically, I've deleted the content of C:\users\<serviceaccount>\appdata\local\temp and then also the directories for Plex Server and Plex Service in the Local directory one level up from that. Somehow, removing both of those seems to work but I think I'm not the only one having some trouble after reconfiguration as this chap also has had a few issues: https://forums.plex.tv/t/plex-crashes-immediately-after-start-httpserver-error-binding-acceptor-an-attempt-was-made-to-access-a-socket-in-a-way-forbidden-by-its-access-permissions/371245. Thanks for coming back to me though and for the great solution to a problem which Plex themselves should be addressing!
